python_modules/dagster/dagster/_daemon/monitoring/monitoring_daemon.py

```diff
@@ -196,6 +210,8 @@ def execute_monitoring_iteration(
             ):
                 monitor_canceling_run(instance, run_record, logger)
                 pass
+            elif run_record.dagster_run.status == DagsterRunStatus.CANCELED:
+                monitor_canceled_run(instance, run_record, logger)
```

Comment:
I don't see how we will ever see runs with this status given that 

`statuses=IN_PROGRESS_RUN_STATUSES + [DagsterRunStatus.CANCELING]`

on line 187 does not include `CANCELED`